web/src/lib/components/NavTab.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    const dispatch = createEventDispatcher();
+    export let items;
+    export let activeItem;
+</script>
+
+<div class="tabs">
+    <ul>
+        {#each items as item}
+            <li on:click={() => dispatch('tabChange', item)}>
+                <div class:active={item === activeItem}>{item}</div>
+            </li>
+        {/each}
+    </ul>
+</div>
+
+<style>
+    .tabs {
```

Comment:
since we went with tailwind css all over the place, could we have this as tailwind classes directly on the elements as well?